test/feat663-search.sh

```diff
@@ -7,6 +7,9 @@ fi
 if ${DUB} search nonexistent123456789package 2>/dev/null; then
     die $LINENO '`dub search nonexistent123456789package` succeeded'
 fi
-if ! OUTPUT=$(${DUB} search dub -v 2>&1) || ! { echo "$OUTPUT" | grep -q '^dub (.*)\s'; } then
+if ! OUTPUT=$(${DUB} search dub -v 2>&1); then
     die $LINENO '`dub search dub` failed' "$OUTPUT"
 fi
+if ! grep -q '^dub (.*)\s'<<<"$OUTPUT"; then
```

Comment:
Here strings are Bash-specific, but I assume there's no one running the testsuite with something other than bash?

Reply:
Shouldn't be the case. There are certainly some other places as well where bash-specifc syntax is used.